spark/src/main/scala/ai/chronon/spark/Driver.scala

```diff
@@ -675,10 +677,17 @@ object Driver {
           if (args.`type`() == "join-stats") {
             fetchStats(args, objectMapper, keyMap, fetcher)
           } else {
+            val featureName = if (args.name.isDefined) {
+              args.name()
+            } else {
+              args.confPath().split("/").takeRight(3).mkString("/")
```

Comment:
`takeRight(3)` might be error-prone if we update the config path. how about putting this logic in a centralized place like a helper method?

Reply:
refactored and done.